app/src/main/res/values/style.xml

```diff
@@ -371,11 +371,11 @@
     </style>
 
     <style parent="Theme.AMOLED_lighter" name="pink_AMOLED_lighter">
-        <item name="colorAccent">@color/md_pink_A200</item>
+        <item name="colorAccent">@color/md_pink_A400</item>
```

Comment:
You're using `md_pink_A400` here. Yet, it isn't defined in colors. This is breaking the build; (you can see the status of the build at the bottom of this PR).